index.html

```diff
@@ -49,177 +51,197 @@ <h1 class="intro-title">I’am a Software Developer</h1>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            </div>
-            <div class="job-tech-lang">
-                <ul class="job-tech-lang-ul">
-                    <li class="job-item job-one">html</li>
-                    <li class="job-item job-one">bootstrap</li>
-                    <li class="job-item job-one">Ruby</li>
-                </ul>
-            </div>
-        </div>
-        <button type="button" class="see-jobs-btn">See Project</button>
-        <div class="job-item-two">
-            <h1 class="job-post-title">Profesional Art Printing Data</h1>
-            <div class="job-post">
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            </div>
-            <div class="job-tech-lang">
-                <ul class="job-tech-lang-ul">
-                    <li class="job-item job-one">html</li>
-                    <li class="job-item job-one">bootstrap</li>
-                    <li class="job-item job-one">Ruby</li>
-                </ul>
-            </div>
-        </div>
-        <button type="button" class="see-jobs-btn">See Project</button>
-        <div class="job-item-two">
-            <h1 class="job-post-title">Profesional Art Printing Data</h1>
-            <div class="job-post">
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            </div>
-            <div class="job-tech-lang">
-                <ul class="job-tech-lang-ul">
-                    <li class="job-item job-one">html</li>
-                    <li class="job-item job-one">bootstrap</li>
-                    <li class="job-item job-one">Ruby</li>
-                </ul>
-            </div>
-        </div>
-        <button type="button" class="see-jobs-btn">See Project</button>
-        <div class="job-item-two">
-            <h1 class="job-post-title">Profesional Art Printing Data</h1>
-            <div class="job-post">
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            </div>
-            <div class="job-tech-lang">
-                <ul class="job-tech-lang-ul">
-                    <li class="job-item job-one">html</li>
-                    <li class="job-item job-one">bootstrap</li>
-                    <li class="job-item job-one">Ruby</li>
-                </ul>
+            <div class="multi-post-inner">
+                <h1 class="post-title">Multi-Post Stories</h1>
+                <div class="post">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
- [ ] It is advisable that your sentence only takes up about 3-4 lines. In your case it's up to five, please mend that. You can improve on it by either reducing the font size slightly or increasing the section's width slightly.